### PR TITLE
Update Pet Info for Dolphin

### PR DESCRIPTION
**Pull request by Enochen**

### CODE REVIEW

**Comment by Enochen** [comment]
Dolphin was changed in a commit 3 days ago

https://github.com/IdleLands/Maps/commit/1ab83f08d2ac2ea17ae1c0fdac6b0a7e5b787389

File: docs/PETS.md [hunk 1]
```diff
@@ -368,7 +368,7 @@ Unlocked by pursuing the physical arts.
 * Smashed Mace
 
 #### Bosses
-* Cloud Giant
+* Butt Giant
```
Comment (seiyria) [inline docs/PETS.md]: uhhh... well I know what extension you have installed.

Comment (Enochen) [inline docs/PETS.md]: Haha whoops, didn't notice that it changed text within the editor

**Comment by seiyria** [comment]
Thanks.

**Pull request merged**